scripts/generators/es_template.py

```diff
@@ -286,30 +269,3 @@ def default_mapping_settings():
             }
         ]
     }
-
-
-def es6_type_fallback(mappings):
-    """
-    Visits each leaf in mappings object and fallback to an
-    Elasticsearch 6.x supported type.
-
-    Since a field like `wildcard` won't have the same defaults as
-    a `keyword` field, we must add any missing defaults.
-    """
-
-    for (name, details) in mappings.items():
-        if 'type' in details:
-            fallback_type = TYPE_FALLBACKS.get(details['type'])
```

Comment:
Is anything still using `TYPE_FALLBACKS`?

If not, that global can be cleared out.